### DISCUSSION TITLE
close tab / clear playlist

### DISCUSSION THREAD

**Discussion by nicbre85** [DeaDBeeF-Player/deadbeef#2852] [General]
Hello, is it possible to have a button to close a tab? or just to clear the playlist tab.

this player is awesome by the way!

**Comment by Oleksiy-Yakovenko** [comment]
What specifically do you mean by a button? Each tab has a button to close it, and also there's a default keyboard shortcut, which can be customized.

**Comment by nicbre85** [comment]
i don't have a button to close the playlists
![Istantanea_2022-12-20_13-53-27](https://user-images.githubusercontent.com/55213484/208672295-7dabb160-d207-4309-85c0-7c991b2f21ea.png)
